385/Assets/Scripts/RopeSystem.cs

```diff
@@ -114,6 +114,15 @@ public bool IsRopeConnected()
 		return (RopeAnchorPoint != null);
     }
 
+    // gets anchor point rope is attached to
+    public Rigidbody2D getAnchor()

```

Comment:
Also take a look at the messages that show in the error list.
![image](https://user-images.githubusercontent.com/16418643/39564487-4df113c0-4e69-11e8-8632-a5652d79ae8e.png)